code/lib/manager-api/src/lib/stories.ts

```diff
@@ -66,7 +66,6 @@ export const transformSetStoriesStoryDataToPreparedStoryIndex = (
       if (docsOnly) {
```

Comment:
Will `docsOnly` exist anymore after we get rid of `.stories.mdx` support @tmeasday ?

Reply:
No, but this code is back compat for composition, is it not @shilman ?